scanpy/readwrite.py

```diff
@@ -913,36 +913,26 @@ def _get_filename_from_key(key, ext=None) -> Path:
 
 
 def _download(url: str, path: Path):
-    try:
-        import ipywidgets
-        from tqdm.auto import tqdm
-    except ModuleNotFoundError:
-        from tqdm import tqdm
-    from urllib.request import urlopen, Request
-
-    blocksize = 1024 * 8
-    blocknum = 0
+    from tqdm.auto import tqdm
+    import requests
```

Comment:
I'd rather keep the previous code that doesn't use requests, just because it's worked for a while, and "if it ain't broke don't fix it".

Reply:
Ok, also reverted, just thought that using `requests` is a bit cleaner.